src/app/core/config/config-fix.json

```diff
@@ -161,6 +163,53 @@
       "isMeeting": true
     }
   ],
+
+  "enum:attendance-status": [
+    {
+      "id": "PRESENT",
+      "shortlabel": "P",
+      "label": "Present",
+      "style": "attendance-P",
+      "countAs": "PRESENT"
+    },
+    {
+      "id": "ABSENT",
+      "shortlabel": "A",
+      "label": "Absent",
+      "style": "attendance-A",
+      "countAs": "ABSENT"
+    },
+    {
+      "id": "LATE",
+      "shortlabel": "L",
+      "label": "Late",
+      "style": "attendance-L",
+      "countAs": "PRESENT"
```

Comment:
I am not so sure if `countAs` is at the right place here. 
I can see the use case here but I am worried that in future requirements, multiple `countAs`  fields would be required for different calculations. However I think this change can be done latter without much additional work.